config/dashboard/kano-jupyter-strings.app

```diff
@@ -0,0 +1,13 @@
+{
+    "title": "Jupyter: Strings",
+    "tagline": "Jupyter Notebook Frontend",
+    "description": "Are you ready to interact with Python on a Browser? Hit Kano Jupyter and enjoy live coding!",
+    "slug": "jupyter",
+    "icon": "kano-jupyter-strings",
+    "colour": "#4A5152",
+    "categories": ["code"],
+    "packages": [],
+    "dependencies": ["kano-jupyter", "kano-profile"],
+    "launch_command": "kano-tracker-ctl session run 'kano-jupyter' 'kano-jupyter run --notebook=strings/Fun.ipynb'",
```

Comment:
Might want to distinguish the different notebooks from a tracking perspective. Maybe we shouldn't worry about it now until we have final versions of each of these.

Reply:
Great idea, fixed that to distinguish tracking